app/views/admin/api_submission/_form.html.erb

```diff
@@ -36,7 +36,7 @@
     width: 'two-thirds',
     id: settings_field_id(f, :service_output_json_key, deployment_environment),
     hint: {
-      text: 'This is the 16 characters key'
+      text: 'This is the 17 characters key'
```

Comment:
Maybe the hint shouldn't specify a length? Seems the key could be any length as defined by the target API